whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierNrtm.java

```diff
@@ -66,14 +65,14 @@ public RpslObject dummify(final int version, final RpslObject rpslObject) {
         return new RpslObject(rpslObject, attributes);
     }
 
-    private void stripSomeNonMandatoryAttributes(List<RpslAttribute> attributes, ObjectType objectType) {
-        if (!STRIPPED_OBJECT_TYPES.contains(objectType)) {
+    private void stripSomeNonMandatoryAttributes(final List<RpslAttribute> attributes, final ObjectType objectType) {
+        if (!ATTRIBUTES_TO_KEEP_OBJECT_TYPE.containsKey(objectType)) {
```

Comment:
What happened to STRIPPED_OBJECT_TYPES ?

Reply:
I think that the idea is to merge two lists into one.
Anyway I think that we should remove STRIPPED_OBJECT_TYPES it not longer used with the last changes.

Reply:
Yes, Initially it was two variables. STRIPPED_OBJECT_TYPES and ATTRIBUTES_TO_KEEP. Since both are being used together I merged them